src/public/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="style.css">
+    <h1> Yoooooo </h1>
+  </head>
+    <body>
+      <h2> Y'all ready for this?</h2>
```

Comment:
Actually, I just noticed you did start with `<h1>`; the problem is that you put it in your `<head>` section. It can't go there. The head section should only contain metadata about the web page, not actual content. The content goes in the body section. You _can_ use a `<header>` tag inside the body section though.